pubspec.yaml

```diff
@@ -129,6 +129,8 @@ dependencies:
   open_file: ^3.3.2
   mime: ^1.0.4
   any_link_preview: ^3.0.0
+  async: ^2.11.0
+  cached_network_image: ^3.2.3
```

Comment:
what is it?

Reply:
![Screenshot 2023-08-09 at 16 36 16](https://github.com/linagora/twake-on-matrix/assets/12546908/e29f91e6-cf03-43e3-b916-3677ace2aa63)
These are fix for that warnings